src/OrchardCore/OrchardCore.Logging.Serilog/WebHostBuilderExtensions.cs

```diff
@@ -5,13 +5,6 @@ namespace OrchardCore.Logging
 {
     public static class WebHostBuilderExtensions
     {
-        public static IHostBuilder UseSerilogWeb(this IHostBuilder builder)
-        {
-            return builder.UseSerilog((hostingContext, configBuilder) =>
-            {
-                configBuilder.ReadFrom.Configuration(hostingContext.Configuration)
-                .Enrich.FromLogContext();
-            });
```

Comment:
Why should this be removed? It could be a single line BTW (and an expression-bodied lambda).

Reply:
The Serilog API has been deprecated that's why I replace it with expression bodies :)

Reply:
The original code wasn't using the deprecated API either.

Reply:
Please check the breaking changes in https://github.com/serilog/serilog-aspnetcore/releases/tag/v8.0.0

Reply:
I checked. This block of code doesn't use the deprecated API. `UseSerilog()` here is not the deprecated `IWebHostBuilder` extension but the still supported `IHostBuilder` one.

Reply:
Please refer to #14697

Reply:
Please elaborate.

Reply:
Strange!! seems it works, @agriffard how did the checks fail in your previous PR?!!

https://github.com/OrchardCMS/OrchardCore/pull/14697/files